### ISSUE TITLE
[BITV] 9.1.1.1a/9.1 - When switching the page to German language, the label of the plus button to create a new group remains in English - "Create group". The tooltip label also remains in English. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41821]
**_Need info from Michael_**

![image](https://github.com/nextcloud/server/assets/6078378/850427ff-2b99-40d6-b968-058e130f7353)

![image](https://github.com/nextcloud/server/assets/6078378/3762ee59-adbe-4688-9a8e-3a5af2481e4c)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-cd239865a5-v9-n1

**Comment by ShGKme** [comment]
A missing translation for `Deutsch (Persönlich: Du)`

**Comment by JuliaKirschenheuter** [comment]
Closing an issue, not relevant
